### ISSUE TITLE
BUG: CI instability (upstream change)

### ISSUE THREAD

**Issue by neutrinoceros** [yt-project/yt#4241]
### Bug report

**Bug summary**

Last night's bleeding edge CI failed:
https://github.com/yt-project/yt/actions/runs/3626064400/jobs/6114722912

I'll dig into this later today

**Comment by neutrinoceros** [comment]
the error can be reproduced by installing numpy from source
```
python -m pip install git+https://github.com/numpy/numpy.git
pytest yt/visualization/tests/test_set_zlim.py::test_float_vmin_then_set_unit
```

**Comment by neutrinoceros** [comment]
The issue is that numpy just broke unyt

Here's a minimal reproducer
```python
import unyt as u
a = 1*u.cm
a in (None, "min")
```

It's not yet clear if this needs a fix in numpy or in unyt. I'll close this ticket once I'm ready to report upstream

**Comment by neutrinoceros** [comment]
fixed with the release of unyt 2.9.3

**Comment by neutrinoceros** [comment]
Turns out the problem still needs to be addressed in yt:
the new version of unyt now correctly lets some, previously hidden, deprecation warnings from numpy surface.
see https://github.com/yt-project/yt/actions/runs/3641878775/jobs/6148337575
